processing/src/main/java/org/apache/druid/query/groupby/GroupByQueryQueryToolChest.java

```diff
@@ -465,9 +513,9 @@ public String apply(DimensionSpec input)
   }
 
   @Override
-  public CacheStrategy<Row, Object, GroupByQuery> getCacheStrategy(final GroupByQuery query)
+  public CacheStrategy<ResultRow, Object, GroupByQuery> getCacheStrategy(final GroupByQuery query)
   {
-    return new CacheStrategy<Row, Object, GroupByQuery>()
+    return new CacheStrategy<ResultRow, Object, GroupByQuery>()
     {
       private static final byte CACHE_STRATEGY_VERSION = 0x1;
```

Comment:
Should this version be bumped so off-machine caches don't cause an issue or will the fancy decorated mapper be used in like `CachingQueryRunner` for reading the values from the cache?

Reply:
Check out the `prepareForCache` method -- the cache format is the same. It was always array-based, and it remains so.

Reply:
Oops, looking closer I see this now, thanks :+1: